config/runtimes/kustomization.yaml

```diff
@@ -1,14 +1,68 @@
 resources:
   - kserve-tensorflow-serving.yaml
+  - kserve-tensorflow-serving-gpu.yaml
   - kserve-sklearnserver.yaml
+  - kserve-mlserver.yaml
+  - kserve-xgbserver.yaml
+  - kserve-tritonserver.yaml
+  - kserve-pmmlserver.yaml
+  - kserve-paddleserver.yaml
+  - kserve-lgbserver.yaml
+  - kserve-pytorchserver.yaml
+  - kserve-pytorchserver-gpu.yaml
+  - kserve-torchserve.yaml
+  - kserve-torchserve-gpu.yaml
 
 images:
   # SMS Only Runtimes
   - name: tensorflow-serving
     newName: tensorflow/serving
     newTag: 1.14.0
 
+  - name: tensorflow-serving-gpu
+    newName: tensorflow/serving
+    newTag: 1.14.0-gpu
+
   - name: kserve-sklearnserver
     newName: kserve/sklearnserver
     newTag: latest
 
+  - name: mlserver
+    newName: docker.io/seldonio/mlserver
+    newTag: 0.2.1
```

Comment:
Looks like MLServer was updated to `0.5.3` via https://github.com/kserve/kserve/commit/f90c669bdc0324aed5b3d527c53c7562858c2dcc . Can you rebase and update the version here?